File: src/api/java/IntExpr.java
/**
Copyright (c) 2012-2014 Microsoft Corporation
   
Module Name:

    IntExpr.java

Abstract:

Author:

    @author Christoph Wintersteiger (cwinter) 2012-03-15

Notes:
    
**/ 

package com.microsoft.z3;

/**
 * Int expressions
 **/
public class IntExpr extends ArithExpr<IntSort>
{
    /**
     * Constructor for IntExpr
     * @throws Z3Exception on error
     **/
    IntExpr(Context ctx, long obj)
    {
        super(ctx, obj);
    }
}

File: src/api/dotnet/IntExpr.cs
/*++
Copyright (c) 2012 Microsoft Corporation

Module Name:

    IntExpr.cs

Abstract:

    Z3 Managed API: Int Expressions

Author:

    Christoph Wintersteiger (cwinter) 2012-11-23

Notes:
    
--*/
using System.Diagnostics;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;


namespace Microsoft.Z3
{
    /// <summary>
    /// Int expressions
    /// </summary>
    public class IntExpr : ArithExpr
    {
        #region Internal
        /// <summary> Constructor for IntExpr </summary>
        internal IntExpr(Context ctx, IntPtr obj)
            : base(ctx, obj)
        {
            Debug.Assert(ctx != null);
        }
        #endregion
    }
}
